collectors/0/kafka_consumer_jmx.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+import re
+import sys
+
+from collectors.lib.jmx_monitor import JmxMonitor
+
+MONITORED_MBEANS = ["kafka.consumer", ""]
+
+
+class KafkaConsumerJmxMonitor(JmxMonitor):
+    USER = "flume"
+    PROCESS_NAME = "org.apache.flume.node.Application"
+    METRIC_PREFIX = "flume"
```

Comment:
We already have that for the broker metrics - the main reason I kept these separate is that, for some reason, the consumer-side Flume hosts actually emit all three sets of metrics (although 2 sets are 0 all the time, obviously). Therefore, I wanted to separate out the consumer and producer metric logic. Let me know if there's a cleaner way to do this.